README.md

```diff
@@ -1,11 +1,28 @@
+<p align="center">
```

Comment:
Apologies that this comment is in the wrong place in the file - at the bottom of the README the license link is relative so it currently won't work in the jazzy docs.